src/robot/utils/text.py

```diff
@@ -27,6 +27,9 @@
 _ERROR_CUT_EXPLN = '    [ Message content over the limit has been removed. ]'
 _TAGS_RE = re.compile(r'\s*tags:(.*)', re.IGNORECASE)
 
+def set_max_error_lines(limit):
+    global _MAX_ERROR_LINES
+    _MAX_ERROR_LINES = limit
```

Comment:
This helper isn't really needed. Better to just rename the constant to `MAX_ERROR_LINES` and let external code to set the module global value directly like

```python
from robot.utils import text

text.MAX_ERROR_LINES
```